openpype/hosts/houdini/api/pipeline.py

```diff
@@ -309,6 +310,7 @@ def _set_context_settings():
         fps
         resolution
         renderer
+        shelves
 
```

Comment:
```suggestion
        renderer

```

The `_set_context_settings` doesn't seem to do anything with shelves so adding it to its docstring seems incorrect?